src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -52,6 +52,12 @@
     <Compile Include="System\IO\Path.cs" />
     <Compile Include="System\Net\WebUtility.cs" />
     <Compile Include="System\Runtime\Versioning\FrameworkName.cs" />
+    <Compile Include="System\Runtime\Versioning\ComponentGuaranteesAttribute.cs" />
+    <Compile Include="System\Runtime\Versioning\ResourceConsumptionAttribute.cs" />
+    <Compile Include="System\Runtime\Versioning\ComponentGuaranteesOptions.cs" />
+    <Compile Include="System\Runtime\Versioning\ResourceExposureAttribute.cs" />
+    <Compile Include="System\Runtime\Versioning\ResourceScope.cs" />
+   <Compile Include="System\Runtime\Versioning\VersioningHelper.cs" />
```

Comment:
Nit: I didn't notice it before, but it looks like this line is missing a single space at the beginning.